src/System.Windows.Forms/src/System/Windows/Forms/PropertyGridInternal/GridEntry.GridEntryAccessibleObject.cs

```diff
@@ -102,11 +100,11 @@ internal override UiaCore.IRawElementProviderFragment FragmentNavigate(UiaCore.N
             /// <summary>
             ///  Return the element that is the root node of this fragment of UI.
             /// </summary>
-            internal override UiaCore.IRawElementProviderFragmentRoot FragmentRoot
+            internal override UiaCore.IRawElementProviderFragmentRoot? FragmentRoot
             {
                 get
                 {
-                    return (PropertyGridView.PropertyGridViewAccessibleObject)Parent;
+                    return Parent is null ? null : (PropertyGridView.PropertyGridViewAccessibleObject)Parent;
```

Comment:
This looks awkward. I'm surprised the original version wasn't good enough....
How about this?
```suggestion
                    return Parent as PropertyGridView.PropertyGridViewAccessibleObject;
```

Reply:
Thank you. The original code did not pass verification after enabling the NRT

Reply:
Still not fixed in the correct commit.

Reply:
Moved to [Enable NRT](https://github.com/dotnet/winforms/pull/4700/commits/86c536b62a85b67be4e21d48b770bd88644934fe) commit